compiler/rustc_session/src/config.rs

```diff
@@ -2435,6 +2435,11 @@ crate mod dep_tracking {
     use std::num::NonZeroUsize;
     use std::path::PathBuf;
 
+    /// Hashes a value for 'dependency tracking' purposes.
+    /// This trait has two uses - computing the crate hash, and
```

Comment:
Can you make this something like `This trait has two uses - as part of crate hash computation and for checking the validity of the incremental compilation on-disk dep-graph and caches.`